### ISSUE TITLE
edge release putting parts directories inside project

### ISSUE THREAD

**Issue by lengau** [canonical/charmcraft#1475]
### Bug Description

Parts directories in charmcraft's current main get placed in the project directory rather than just in the provider

### To Reproduce

pack a charm with edge

### Environment

n/a

### charmcraft.yaml

```shell
n/a
```

### Relevant log output

```shell
https://matrix.to/#/!LhFxJIPEcCacgdMghH:ubuntu.com/$CwAAq7aXat08QB-wecEVf2SzZp4NongaVgB52N2TFjg?via=ubuntu.com&via=matrix.org
```

**Comment by dimaqq** [comment]
I imagine there's a new edge release by now, did the linked commits make it to the main branch or are those stuck in some PR?

**Comment by lengau** [comment]
Yeah this is fixed with 3.1 (possibly earlier).
